test/rosdep_repo_check/apk.py

```diff
@@ -93,6 +114,11 @@ def enumerate_apk_packages(base_url, os_name, os_code_name, os_arch):
                 pkg_url = os.path.join(base_url, pkg_filename)
                 yield PackageEntry(pkg_name, pkg_version, pkg_url, source_name=source_name)
 
+                if 'p' in index_entry:
+                    for d in parse_deps(index_entry['p']):
+                        if d.type is None:
```

Comment:
Could you elaborate on what `type` represents and why we'd skip avoid aliases with one?

Reply:
Added some docstrings.
https://github.com/alpine-ros/rosdistro/blob/79e2e6dab0952cf647614da798b224e2e7addd22/test/rosdep_repo_check/apk.py#L70-L77
> the type of the Dependency.
> e.g.
> - None: package
> - 'cmd': command
> - 'so': shared object